modules/dxp/apps/search-experiences/search-experiences-test/src/testIntegration/java/com/liferay/search/experiences/internal/model/listener/test/CompanyModelListenerTest.java

```diff
@@ -43,8 +46,13 @@ public class CompanyModelListenerTest {
 
 	@Test
 	public void testSXPElements() throws Exception {
+		Bundle bundle = FrameworkUtil.getBundle(_modelListener.getClass());
```

Comment:
@MrJohn1911 Please rename this test and move it to proper package, others look good to me, thanks!